apps/condo/domains/marketplace/components/MarketplaceOnboarding/RequisitesSetup.tsx

```diff
@@ -137,25 +143,40 @@ export const RequisitesSetup: React.FC = () => {
         if (!acquiringLoading && !acquiringError && !!acquiringId) {
             setError(null)
             setIsLoading(true)
-            createAction({
-                integration: { connect: { id: acquiringId } },
-                status: INVOICE_CONTEXT_STATUS_INPROGRESS,
-                organization: { connect: { id: orgId } },
-                recipient: {
-                    tin: get(organization, 'tin'),
-                    bic: values.bic,
-                    bankAccount: values.account,
-                },
-                taxRegime: values.taxType,
-                vatPercent: String(values.taxPercent),
-                currencyCode: 'RUB',
-            }).then(() => {
+            let promise: Promise<InvoiceContext>
```

Comment:
Promise is to general name for this variable
```suggestion
            let invoiceCreateOrUpdate: Promise<InvoiceContext>
```